go/oprf/oprf_test.go

```diff
@@ -52,6 +54,10 @@ func TestFullVOPRFP521Multiple(t *testing.T) {
 	checkFull(t, validOPRFP521Ciphersuite, 5)
 }
 
+func TestFullCurve448(t *testing.T) {
+	checkFull(t, validOPRFCURVE448Ciphersuite, 1)
```

Comment:
Also need to add tests for the verifiable ciphersuite: `validVOPRFCURVE448Ciphersuite` and for cases where there are multiple inputs.